parcels/compilation/codegenerator.py

```diff
@@ -819,6 +826,16 @@ def visit_FieldEvalNode(self, node):
         self.visit(node.field)
         self.visit(node.args)
         args = self._check_FieldSamplingArguments(node.args.ccode)
+        statements_croco = []
+        if "croco" in node.field.obj.gridindexingtype and node.field.obj.name != "H":  # TODO needs to be sigma
```

Comment:
is this TODO still current?

Reply:
No, was a stray comment. Removed the TODO